src/rgw/rgw_process.cc

```diff
@@ -297,7 +298,19 @@ int process_request(rgw::sal::RGWRadosStore* const store,
         //                                       frontend_prefix,
         //                                       client_io, &mgr, &init_error);
         //op = handler->get_op();
+
         ret = op->verify_requester(auth_registry);
+
+        if(backup_range)
```

Comment:
please use a brace/compound statement style consistent with (modern updates to) these files